contracts/GaugeController.sol

```diff
@@ -33,16 +79,43 @@ contract GaugeController is IGaugeController {
         rewardVault = _rewardVault;
     }
 
+    modifier requireGauge(address _gauge) {
+        require(isGauge(_gauge), "GaugeController:invalid-address");
```

Comment:
We should also create an internal function `_isGauge` to call in this modifier to avoid calling a public function.

Reply:
Agree. Out of scope for natspec PR though.